packages/nimble-components/package.json

```
@@ -12,8 +12,8 @@
     "prettier-fix": "prettier-eslint \"**/*.*\" --write --prettier-ignore",
     "pack": "npm pack",
     "invoke-publish": "npm publish",
-    "storybook": "start-storybook -p 6006",
-    "build-storybook": "build-storybook -o dist/storybook --webpack-stats-json",
+    "storybook": "storybook dev -p 6006",
+    "build-storybook": "storybook build -o dist/storybook",
```

Likely still necessary as we pass a [prebuilt Storybook to Chromatic](https://www.chromatic.com/docs/turbosnap#prebuilt-storybook)
```suggestion
    "build-storybook": "storybook build -o dist/storybook --webpack-stats-json",
```